assets/wizards/site-design/index.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * Site Design
+ */
+
+/**
+ * WordPress dependencies.
+ */
+import { Component, render, Fragment } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
```

Comment:
`apiFetch` isn't in use - this can be removed.

Reply:
Done in https://github.com/Automattic/newspack-plugin/pull/421/commits/83a97b9c9f4acb80bfe55b0d244a841910212256